scripts/quests/windurst/Acting_in_Good_Faith.lua

```diff
@@ -86,8 +89,10 @@ quest.sections =
             onEventFinish =
             {
                 [50] = function(player, csid, option, npc)
-                    player:messageSpecial(eldiemeID.text.SPIRIT_INCENSE_EMITS_PUTRID_ODOR, xi.ki.SPIRIT_INCENSE)
+                    player:messageSpecial(ID.text.SPIRIT_INCENSE_EMITS_PUTRID_ODOR, xi.ki.SPIRIT_INCENSE)
                     player:delKeyItem(xi.ki.SPIRIT_INCENSE)
+
+                    npc:setPos(unpack(ID.npc.QM1_POS[math.random(1, 4)])) -- move qm1 randomly on quest completion
```

Comment:
`ID` references nothing. `eldiemeID`

Reply:
Switched eldiemeID to just ID if that's alright 😬

Reply:
I'm sorry, but its not. revert it to how it was.

Reply:
Done ✅